code/lib/test/package.json

```diff
@@ -68,7 +68,6 @@
       "./src/index.ts"
     ],
     "noExternal": [
-      "@testing-library/dom",
```

Comment:
Using @storybook/test inside of nextjs had a very weird result where an internal library which is part of @testing-librayr/dom, `lz-string`, ends up completely ovewriting the module export of @storybook/test, so instead of that export containing things like `within, fn, etc.`, it only had the contents of `lz-string`. Making this change fixed the problem, but I don't know the implications of it! cc @kasperpeulen

Reply:
Hm let's debug that together. We prebundle it because of performance.